CHANGELOG.md

```diff
@@ -30,6 +30,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Update dependencies;
 - Return `NetworkInfoDto` instead of `NetworkInfo` in message_interface;
 
+## 2.0.1-rc.2 - 2022-09-29
```

Comment:
This should replace `## Unreleased - YYYY-MM-DD` in L22 and the other things should merge

Reply:
Ah, sorry my bad. Please take another look :)